ingestion/functions/test_clean_old_ingestion_source_files.py

```diff
@@ -0,0 +1,77 @@
+import boto3
+import os
+import pytest
+import unittest
+from datetime import datetime, timedelta
+from botocore.exceptions import ClientError
```

Comment:
Nitpicking: third-party imports should live in their own group between standard library imports and local application imports